src/test/javascript/spec/component/code-editor/code-editor-ace.component.spec.ts

```diff
@@ -48,21 +51,17 @@ describe('CodeEditorAceComponent', () => {
                 debugElement = fixture.debugElement;
                 codeEditorRepositoryFileService = debugElement.injector.get(CodeEditorRepositoryFileService);
                 loadRepositoryFileStub = jest.spyOn(codeEditorRepositoryFileService, 'getFile');
+                getInlineFeedbackNodeStub = jest.spyOn(comp, 'getInlineFeedbackNode');
+                getInlineFeedbackNodeStub.mockReturnValue(document.createElement('div'));
+                // Mock the ResizeObserver, which is not available in the test environment
+                global.ResizeObserver = jest.fn().mockImplementation((...args) => new MockResizeObserver(args));
             });
     });
 
     afterEach(() => {
         jest.restoreAllMocks();
     });
 
-    it('should know the lines in which inline feedback is shown', () => {
```

Comment:
It was testing internal behavior. Whenever feedback changed, we previously updated the arrays that are tested here. This is not needed anymore because we now directly calculate this with a simple function (`filterFeedbackForFile`) that is indirectly tested through the other parts of the component.